README.md

```diff
@@ -62,4 +62,53 @@ This will start a local development server on port 5000 (http://localhost:5000).
 This project welcomes contributors. If you are interested, please feel free to
 join [the mailing list](https://mail.mozilla.org/listinfo/bugsahoy-devel)
 
+## Add a project
```

Comment:
Put this before the Contributing section. I would also rename to `## Adding a project`